drms/utils.py

```diff
@@ -72,10 +72,11 @@ def to_datetime(tstr, force=False):
     result : pandas.Series or pandas.Timestamp
         Pandas series or a single Timestamp object.
     """
-    s = pd.Series(tstr)
+    s = pd.Series(tstr).astype(str)
     if force or s.str.endswith('_TAI').any():
         s = s.str.replace('_TAI', '')
         s = s.str.replace('_', ' ')
         s = s.str.replace('.', '-', n=2)
     res = _pd_to_datetime_coerce(s)
-    return res.iloc[0] if isinstance(tstr, six.string_types) else res
+    valid_types = (six.string_types, six.integer_types, float)
```

Comment:
Actually, only strings are supported to give meaningful results. The test in the end was used to check if the input was a single string and return a single timestamp in this case, not a Series.

Now that non-string values should get handled correctly (resulting in NaT), a better way would be to use `numpy.isscalar()` instead. What do you think about this?
```python
return res.iloc[0] if (len(res) == 1) and np.isscalar(tstr) else res
```